sdk/identity/azure-identity/tests/test_shared_cache_credential.py

```diff
@@ -354,9 +345,7 @@ def test_same_username_different_tenants():
         credential.get_token("scope")
     # error message should indicate multiple matching accounts, and list discovered accounts
```

Comment:
This comment should be updated or removed because we no longer expect the message to list discovered accounts.